### DISCUSSION TITLE
Welcome to MyChatAI-ChatGPT-client Discussions!

### DISCUSSION THREAD

**Discussion by jerryjee120** [jerryjee120/MyChatAI-ChatGPT-client#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pioner22** [comment]
Make it possible to copy the entire AI response with one button, as it is done when the AI prints the code.

**Reply by jerryjee120** [reply]
1. If you use the macOS version, you can copy all by right clicking the message and selecting the copy menu item.
2. There is a bug in mobile version for copy menu. It will be fixed in the next update.

Thanks for you feedback.

**Comment by GeminiEnfield** [comment]
Can we have some more information about the Cloud credits?
I currently use the OpenAI API and I know what the costs are for using that. I'd like to know what the Cloud costs are per point for ChatGPT and DALL-E. And do the unused points rollover at the end of the month?
